packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -58,13 +58,15 @@ const modalKindAndSizeToClassName: Partial<
     [ModalSize.Medium]:
       'mobileL:max-h-[calc(100vh-7.5rem)] max-h-[calc(100vh-2.5rem)] h-auto min-h-[25rem] mt-6',
     [ModalSize.Large]: 'laptop:mt-14 laptop:mb-10',
+    [ModalSize.XLarge]: 'laptop:mt-14 laptop:mb-10',
   },
 };
-const modalSizeToClassName: Record<ModalSize, string> = {
+export const modalSizeToClassName: Record<ModalSize, string> = {
```

Comment:
If the page using this already imports from `modal` you can also add it to the main export here?
It will be one less line.

Reply:
Right. Have updated it now.